packages/lit/src/element/lit-root.ts

```diff
@@ -103,22 +102,22 @@ export class BlockSuiteRoot extends WithDisposable(ShadowlessElement) {
     return this.renderModel(root);
   }
 
-  renderModel = (model: BaseBlockModel): TemplateResult => {
+  renderModel = (model: BaseBlockModel): TemplateResult | null => {
     const { flavour, children } = model;
     const schema = this.page.schema.flavourSchemaMap.get(flavour);
     if (!schema) {
       console.warn(`Cannot find schema for ${flavour}.`);
-      return html`${nothing}`;
+      return null;
```

Comment:
Maybe we should keep `nothing`[^1] here.

[^1]: [`nothing` vs `null`](https://github.com/lit/lit/issues/784#issuecomment-460086850)